docs/configuration/monitoring.md

```diff
@@ -75,20 +106,20 @@ Please see the [README.md](https://github.com/codacy/chart/blob/master/README.md
 
 You **should** provide a password for the `crow` installation either through the `values.yaml` file or through a `--set` parameter during the `helm` installation process. This parameter can be configured as follows:
 
-* Through a `--set` parameter:
+-   Through a `--set` parameter:
```

Comment:
You need to rebase.. there is a new config for remark that forces `*` instead of `-`

Reply:
That's strage. I'm based on `ca2d5bb`, "Sandbox/slack artifact url", and there was only commit after that, which didn't change remark. Codacy also didn't pickup anything. But I'll check.